src/Diagnostics.ModelsAndUtils/Utilities/StringExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Diagnostics.ModelsAndUtils.Utilities
+{
+    public static class StringExtensions
+    {
+        public static string Truncate(this string value, int maxLength)
+        {
+            if (string.IsNullOrEmpty(value))
```

Comment:
string.IsNullOrWhitespace?

Reply:
Whitespace can go over the length limit so it can be truncated, and I'd prefer not to mess with whitespace values if anyone wanted to use this function to truncate them. It would be up to the caller to trim their own whitespace.